### ISSUE TITLE
[Request] Add more slot on Q-table

### ISSUE THREAD

**Issue by lyquocnam** [quasarframework/quasar#578]
Hi author, i need add quick add form to first of tbody of datatable so you can add more slot to qtable ?

**Comment by rstoenescu** [comment]
Hi, Can you rephrase that please. Not sure what you want added. Thanks!

**Comment by lyquocnam** [comment]
![image](https://cloud.githubusercontent.com/assets/8347826/26250627/d778d9b8-3cd4-11e7-9e58-7e481ccb714f.png)
like picture @rstoenescu

**Comment by BenoitRanque** [comment]
There is no slot limit. You can have 1 slot per column. Your issue does not seem related to data-table

**Comment by lyquocnam** [comment]
how can i add custom row with form after header row @BenoitRanque  ? i want to create quick form in data-table

**Comment by lyquocnam** [comment]
dear author, please add this feature to v0.14, this will allow we add custom row (may be form) and do many other things.

**Comment by rstoenescu** [comment]
A revamp of DataTable is scheduled soon, and it will allow a whole lot more customization, including your request. I ask of you to be patient. Thanks.

**Comment by lyquocnam** [comment]
@rstoenescu i'm so exciting :D

**Comment by nueko** [comment]
I think we need a slot beside refresh button, for create button, maybe

**Comment by jrandall16** [comment]
@nueko I agree, I think a slot for the entire title bar would be beneficial. There seems to be no good place to put a create button right now, and having a slot for the title bar would allow for complete customization of the title bar.

**Comment by rstoenescu** [comment]
DataTable revamp is in progress for v0.15. It will include this. Thanks for your patience.

**Comment by rstoenescu** [comment]
Closing this as code is ready in future v0.15 datatable revamp.
